lib/platform-reactnative.js

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+require('setimmediate');
+require('fast-text-encoding');
+const crypto = require('isomorphic-webcrypto');
+
+exports.setImmediate = setImmediate;
+
+// Exported by 'isomorphic-webcrypto'
+exports.crypto = crypto;
+
+// Exported by 'fast-text-encoding'
+exports.TextEncoder = TextEncoder;
```

Comment:
It looks like `fast-text-encoding` is a polyfill of the global?  Does it need to be exported or is the `require` above enough to make it work?